app/src/main/res/values-pt-rBR/strings.xml

```diff
@@ -13,6 +13,7 @@
   <string name="menu_help">Ajuda</string>
   <string name="exploration_activity_title">Reprodutor de Exploração</string>
   <string name="help_activity_title">Ajuda</string>
+  <string name="bottom_sheet_options_menu_close">Fechar</string>
```

Comment:
Please don't add strings for other languages, that's a separate process.

Reply:
Removed